lib/internal/url.js

```diff
@@ -1365,6 +1372,10 @@ function getPathFromURLPosix(url) {
   return decodeURIComponent(pathname);
 }
 
+/**
+ * @param {string | URL} path 
+ * @returns {string}
+ */
```

Comment:
Should we document thrown errors using jsdoc's `@throws` tag?
This function throws `TypeError` in two different circumstances with varying error codes.

Reply:
from a practical perspective I don't think documenting the thrown errors is too useful since with current tooling it isn't  actually able to guard them on a try/catch. In JS , stack depth, out of memory, etc. likely would cause such tags to be incorrect actually.